packages/manager/apps/telecom/src/app/telecom/telephony/line/softphone/addDevice/add-device.controller.js

```diff
@@ -0,0 +1,10 @@
+export default class SoftphoneAddDeviceController {
+  /* @ngInject */
+  constructor() {
+    this.input = { name: null, recordLink: '-' };
```

Comment:
```suggestion
    this.model = { name: null, recordLink: '-' };
```
The record link will be not send to api so you should take it out to your model object.